platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -58,8 +58,8 @@ - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
     if (range.location + range.length > [self pointCount])
     {
         [NSException raise:NSRangeException
-                    format:@"Invalid coordinate range %@ extends beyond current coordinate count of %zu",
-                        NSStringFromRange(range), [self pointCount]];
+                    format:@"Invalid coordinate range %@ extends beyond current coordinate count of %@",
+                        NSStringFromRange(range), @([self pointCount])];
```

Comment:
This isn’t necessary. There should be a fix-it that casts `self.pointCount` to `unsigned long`.